packages/stargate/src/stargateclient.ts

```diff
@@ -223,13 +227,22 @@ export class StargateClient {
           ? await this.txsQuery(`tx.height=${query.height}`)
           : [];
     } else if (isSearchBySentFromOrToQuery(query)) {
-      throw new Error(
-        "This type of search query is not yet implemented. See https://github.com/cosmos/cosmjs/issues/533.",
+      const sentQuery = withFilters(`message.module='bank' AND transfer.sender='${query.sentFromOrTo}'`);
```

Comment:
Does it make sense to search for a transaction by ID but filter by height? Searching by ID is kind of a weird case for `searchTx` because it shouldn't ever return more than one transaction.

Reply:
> Does it make sense to search for a transaction by ID but filter by height?

No, so you better don't add a hight filter. But given the API we have, the current behavior make no sense to me.

> Searching by ID is kind of a weird case for `searchTx` because it shouldn't ever return more than one transaction.

Yes, good point. We should convert this search type to its own endpoint.

Reply:
OK, I'll create a separate issue to cover stargate and launchpad.

Reply:
https://github.com/cosmos/cosmjs/issues/579